### ISSUE TITLE
city/settlement/road/ship draw ordering

### ISSUE THREAD

**Issue by kotc** [jdmonin/JSettlers2#105]
sometimes its ok, sometimes its weird:

![catan-element-draw-ordering](https://github.com/jdmonin/JSettlers2/assets/10637309/58ca97b6-06c8-402d-8b3e-4648497acec2)

![catan-element-draw-ordering-2](https://github.com/jdmonin/JSettlers2/assets/10637309/894653d3-e7a9-44c9-9665-5f6c2837acc2)

**Comment by jdmonin** [comment]
Agreed. I think this may be related to the highlight of the most recently placed piece of current turn
